ls1intum/Artemis
# Fix the layout of the video unit creation form

### Checklist

### Motivation and Context
Before
![grafik](https://user-images.githubusercontent.com/72132281/125823970-be6aa83f-b091-4411-8649-b42e3f4bd221.png)
After
![grafik](https://user-images.githubusercontent.com/72132281/125823983-20ab1751-b615-48db-b244-d5d4786fb228.png)

### Description

`form-row` was removed in Bootstrap 5. We can use just `d-flex` as replacement in our specific case. I also increased the bottom-margin of the conversion arrow by one because it looked better to me.

### Steps for Testing

1. Find or create a lecture
2. Create a video unit for it
3. Check if the creation page looks good (~like in the latter screenshot above)
